x/gov/spec/01_concepts.md

```diff
@@ -140,12 +140,17 @@ casted on a proposal for the result to be valid.
 Threshold is defined as the minimum proportion of `Yes` votes (excluding
 `Abstain` votes) for the proposal to be accepted.
 
-Initially, the threshold is set at 50% with a possibility to veto if more than
-1/3rd of votes (excluding `Abstain` votes) are `NoWithVeto` votes. This means
-that proposals are accepted if the proportion of `Yes` votes (excluding
-`Abstain` votes) at the end of the voting period is superior to 50% and if the
-proportion of `NoWithVeto` votes is inferior to 1/3 (excluding `Abstain`
-votes).
+Initially, the threshold is set at 50% of `Yes` votes, excluding `Abstain`
+votes. A possibility to veto exists if more than 1/3rd of all votes are
```

Comment:
I'd add somewhere (if it's not done alreaady) that these 50% and 1/3 numbers are tweakable by governance, and not hardcoded in the spec.

Reply:
This is a good point!